src/httpServerClass.js

```diff
@@ -0,0 +1,43 @@
+import express from 'express';
+import http from 'http';
+import { once, EventEmitter } from 'events';
+
+export default class HttpServer {
+    constructor () {
+        this.sockets = [];
+        this.server = this.createServer();
+    }
+
+    createServer () {
+        const app = express();
+
+        this.server = http.Server(app);
+
+        app.get('/', (req, res) => {
+            res
+                .status(200)
+                .sendFile('./html/main.html', { root: './' });
```

Comment:
I would like to see a static middleware here, but I'm sure that you already use it in the `tsNodeServer` branch. If not - it's better to think about it in the next PR :)